.github/funding.yml

```diff
@@ -1 +0,0 @@
-github: hsynlms
```

Comment:
Is there any specific reason to remove the file?

Reply:
No reason! Can be replaced 👍